### PR TITLE
Streamline commands for "hammer host create"

### PR DESCRIPTION
**Pull request by maximiliankolb**
* Sort options in alphabetical order
* Use generic examples
* Prefer "--organization" over "--organization-id"
* Use descriptive variable names

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)

### CODE REVIEW

File: guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc [hunk 1]
```diff
@@ -125,7 +134,9 @@ endif::[]
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer bootdisk host --host _My_Host_Name.example.com_ --full true
+# hammer bootdisk host \
+--full true \
+--host _My_Host_Name.example.com_
```
Comment (adamlazik1) [inline guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc:139]:
```suggestion
--host _My_Host_Name_
```

File: guides/common/modules/proc_adding-a-bonded-interface.adoc [hunk 2]
```diff
@@ -39,33 +39,34 @@ endif::[]
...
    --interface="identifier=_bond0_, \
-               ip=_172.25.18.123_, \
+               ip=_My_IP_Address_2_, \
                type=Nic::Bond, \
                mode=active-backup, \
-               attached_devices=[_eth1,eth2_], \
+               attached_devices=[_My_NIC_1_,_My_NIC_2_], \
                managed=true, \
                domain_id=_My_Domain_ID_, \
                subnet_id=_My_Subnet_ID_" \
```
Comment (adamlazik1) [inline guides/common/modules/proc_adding-a-bonded-interface.adoc:67]: I'd say this probably doesn't need indenting, at least not the first level. What do you think?

File: guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Rebased to master and applied both of your suggestions. Please re-review the last two commits.

**Review by mjivraja — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_creating-network-or-image-based-hosts.adoc [hunk 3]
```diff
@@ -45,37 +45,37 @@ ifeval::["{context}" == "kvm-provisioning"]
...
+--hostgroup "_My_Host_Group_" \
 --interface "managed=true,primary=true,provision=true,compute_type=network,compute_network=_examplenetwork_" \
---compute-attributes="cpus=1,memory=1073741824" \
---volume="pool_name=default,capacity=20G,format_type=qcow2" \
---root-password "_password_"
+--location "_My_Location_" \
+--managed true \
+--name "_My_Host_Name_" \
+--organization "_My_Organization_" \
+--provision-method build \
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-network-or-image-based-hosts.adoc]: Applied.

File: guides/common/modules/proc_creating-network-or-image-based-hosts.adoc [hunk 4]
```diff
@@ -87,17 +87,17 @@ ifeval::["{context}" == "kvm-provisioning"]
...
---managed true \
+--hostgroup "_My_Host_Group_" \
+--image "_My_KVM_Image_" \
 --interface "managed=true,primary=true,provision=true,compute_type=network,compute_network=examplenetwork" \
---compute-attributes="cpus=1,memory=1073741824" \
+--location "_My_Location_" \
+--managed true \
+--name "_My_Host_Name_" \
+--organization "_My_Organization_" \
+--provision-method image \
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-network-or-image-based-hosts.adoc]: Applied.

File: guides/common/modules/proc_creating-network-or-image-based-hosts.adoc [hunk 5]
```diff
@@ -106,17 +106,17 @@ ifeval::["{context}" == "rhv-provisioning"]
...
---managed true \
+--hostgroup "_My_Host_Group_" \
+--image "_My_{oVirtShort}_Image_" \
 --interface "managed=true,primary=true,provision=true,compute_name=eth0,compute_network=satnetwork" \
---compute-attributes="cluster=Default,cores=1,memory=1073741824,start=true" \
+--location "_My_Location_" \
+--managed true \
+--name "_My_Host_Name_" \
+--organization "_My_Organization_" \
+--provision-method image \
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-network-or-image-based-hosts.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Applied all feedback and rebased to master.

**Review by adamlazik1 — APPROVED** [review]
LGTM! how about @mjivraja?

**Review by mjivraja — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   11f9ab842..d53143982  3.8 -> 3.8
   82edd512a..bd2a3d242  3.7 -> 3.7

**Pull request merged**